internal/pkg/externalplugins/cherrypicker/cherrypicker.go

```diff
@@ -254,7 +254,12 @@ func (s *Server) handleIssueComment(l *logrus.Entry, ic github.IssueCommentEvent
 	if len(cherryPickMatches) == 0 || len(cherryPickMatches[0]) != 2 {
 		return nil
 	}
-	targetBranch := strings.TrimSpace(cherryPickMatches[0][1])
+
+	var targetBranches []string
```

Comment:
Maybe we should consider the possible duplication of `targetBranch` and use the set structure.

Reply:
Good catch. Fixed. Thanks!